include/functions.h

```diff
@@ -264,7 +264,7 @@ void __osPackReadData(void);
 uintptr_t osVirtualToPhysical(void* vaddr);
 u32 __osGetSR(void);
 void __osSetSR(u32 value);
-void osWritebackDCache(void* vaddr, s32 nbytes);
+void osWritebackDCache(void* vaddr, size_t nbytes);
```

Comment:
Just a note, this does affect matching in `Load2_LoadOverlay` in that I had to change size to `s32` instead of `size_t`. `Load_LoadOverlay` seems fine to leave as `size_t`.